### PR TITLE
Updates to Location Nova resource

### PR DESCRIPTION
**Pull request by chx2**
Working on improvements to the Location Nova resource:

- [x] Fix Market to only display title + state abbreviation
- [x] Save abbreviation as all uppercase
- [x] Default timezone to Market timezone
- [ ] Add panels for GMB details and hours

Also adding references to Nova resources in this package via https://github.com/tipoff/support/pull/127

### CODE REVIEW

File: src/Nova/Location.php [hunk 1]
```diff
@@ -48,7 +48,9 @@ public function fields(Request $request)
             Text::make('Abbreviation')
                 ->withMeta(['extraAttributes' => ['maxlength' => 4]])
                 ->required(),
-            nova('timezone') ? BelongsTo::make('Timezone', 'timezone', nova('timezone'))->required() : null,
+            nova('timezone') ? BelongsTo::make('Timezone', 'timezone', nova('timezone'))
```
Comment (chx2) [inline src/Nova/Location.php]: Not quite sure if there's a way to auto-update this once a Market is selected, so for now I just made the field optional and added a label that indicates the field will be populated if nothing else is selected.

Screen Shot 2021-04-12 at 8 32 49 AM

Comment (drewroberts) [reply src/Nova/Location.php]: Yeah, that's perfect. Thanks!

**Comment by drewroberts** [comment]
Issue #68

**Comment by drewroberts** [comment]
@chx2 I'm going to go ahead and merge this PR so I can move forward with some of the stuff I'm doing in the tipoff/demo application. You can create the panel for the current GMB data in another PR. Thanks!

**Review by drewroberts — APPROVED** [review]
Thank you! ⛵

**Pull request merged**